### PR TITLE
Soorya | 1206804233224150 | Ward level filter

### PR DESCRIPTION
**Pull request by SooryaKumaranC-tw**
Asana Link: https://app.asana.com/0/1204322126657135/1206804233224150/f

### CODE REVIEW

File: src/features/CareViewPatients/utils/CareViewPatientsUtils.js [hunk 1]
```diff
@@ -3,12 +3,28 @@ import {
...
-    params: { offset, limit },
-  });
+export const fetchPatientsList = async (
+  wardId,
+  offset,
+  limit = 3,
+  headerSelected,
+  providerUuid
+) => {
+  if (headerSelected === "TOTAL_PATIENTS") {
```
Comment (Phanindra-tw) [inline src/features/CareViewPatients/utils/CareViewPatientsUtils.js:16]: Can we push these string literals into the constants folder? I see it is being used in multiple places
`MY_PATIENTS` and `TOTAL_PATIENTS`

Comment (SooryaKumaranC-tw) [reply src/features/CareViewPatients/utils/CareViewPatientsUtils.js:16]: Sure I will do. Thanks

File: src/features/CareViewPatients/styles/CareViewPatients.scss [hunk 2]
```diff
@@ -1,13 +1,26 @@
+@import "src/styles/styles";
 .care-view-patients-container {
   padding: 20px 5%;
   display: flex;
   justify-content: center;
 
   .no-search-results {
-    padding: 20px 0 20px 20px;
-    margin: 10px 135px 10px 420px;
+    // padding: 20px 0 20px 20px;
```
Comment (Phanindra-tw) [inline src/features/CareViewPatients/styles/CareViewPatients.scss:8]: Can we remove this unused css?

Comment (SooryaKumaranC-tw) [reply src/features/CareViewPatients/styles/CareViewPatients.scss:8]: Sure

**Review by Phanindra-tw — APPROVED** [review]

**Pull request merged**